styles/avatar.css

```diff
@@ -0,0 +1,89 @@
+.main-container-components {
+  margin-left: 15rem;
+}
+
+.container-text {
+  width: 15rem;
+}
+
+.heading {
+  margin-left: 1rem;
+  font-size: 2.5rem;
+  font-weight: 500;
+}
+
+.description {
+  padding: 1rem;
+  font-size: 1.2rem;
+}
+
+.heading-avatar {
+  font-size: 2rem;
+}
+
+.description-avatar {
+  font-size: 1.1rem;
+}
+
+.avatar-container {
+  background-color: var(--body-background-color);
+  max-width: 100%;
+  height: 10rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  flex-wrap: wrap;
+}
+
+.avatar {
+  border-radius: 50%;
+  height: 50px;
+  width: 50px;
+}
+
+.avatar-lg {
+  height: 100px;
+  width: 100px;
+}
+
+.avatar-md {
+  height: 75px;
+  width: 75px;
+}
```

Comment:
use relative units

Reply:
This is for special cause !! in order to keep the images in descending order i have done it.